tests/ansible/integration/interpreter_discovery/ansible_2_8_tests.yml

```
@@ -63,15 +63,31 @@
             ping:
             register: legacy
 
-          - name: check for dep warning (only on platforms where auto result is not /usr/bin/python and legacy is)
+            # from ansible 2.12 on this changed
+            #  - https://docs.ansible.com/ansible/devel/porting_guides/porting_guide_5.html#python-interpreter-discovery
+            #  - https://docs.ansible.com/ansible/latest/reference_appendices/interpreter_discovery.html
+            # default discovery methot is now auto and will default to python3
+            # and the message changed from a deprecation warning to a real warning that can not be suppressed by
+            # using deprecation_warnings=False
+          - name: check for dep warning (only on platforms where auto result is not /usr/bin/python and legacy is) for ansible 2.8-2.11
```

This shuffles the comment so code collapse can swallow it, and fixes a minor typo
```suggestion
          - name: check for dep warning (only on platforms where auto result is not /usr/bin/python and legacy is) for ansible 2.8-2.11
            # from ansible 2.12 on this changed
            #  - https://docs.ansible.com/ansible/devel/porting_guides/porting_guide_5.html#python-interpreter-discovery
            #  - https://docs.ansible.com/ansible/latest/reference_appendices/interpreter_discovery.html
            # default discovery method is now auto and will default to python3
            # and the message changed from a deprecation warning to a real warning that can not be suppressed by
            # using deprecation_warnings=False
```